lib/cell-manager.coffee

```diff
@@ -37,3 +27,39 @@ module.exports = CellManager =
         console.log 'CellManager: Cell [start, end]:', [start, end], 'row:', row
 
         return [start, end]
+
+
+    getBreakpoints: ->
+        editor = atom.workspace.getActiveTextEditor()
+        buffer = editor.getBuffer()
+        breakpoints = [0]
+
+        regex = @getRegex editor, 'g'
+        if regex?
+            buffer.scan regex, ({range}) ->
+                breakpoints.push range.start.row
```

Comment:
Again no need to act this comment, but to address the case of inline comments fully, here we may need to store `breakpoints.push range.start`/